### ISSUE TITLE
Waiting while over blankets should apply warmth the same as sleeping.

### ISSUE THREAD

**Issue by RogueYun** [CleverRaven/Cataclysm-DDA#14364]
I have a suggestion.
"You use your pile of clothes for warmth." should also apply to waiting as it does to sleeping. No sense freezing to death waiting on a pile of two blankets, two sheets, and a pair of boxer shorts.

**Comment by sparr** [comment]
waking up has a time penalty. stopping waiting does not. if we allow you to wrap up in blankets while waiting, you'd need to have a "taking off clothing" time penalty at the very least.

a better fix for this is to just make a better system for interrupting activities, including waiting, when cold/pain are causing ongoing damage.

**Comment by Coolthulhu** [comment]
It's less about freezing to death and more about the ability to warm up without having to go to sleep.
If lack of time penalty is a problem, there could be a lower limit below which floor warmth doesn't activate.

Sleeping currently doesn't cause proneness, meaning that the survivor and NPCs sleep standing. If that changes, getting wrapped up in blankets should require proneness and the interface should distinguish between "alert waiting" and "rest waiting", with the former not causing the survivor to lie down.

Reading should probably benefit from floor warmth too. Some specifically selected crafting recipes could to (for example, knitting).

**Comment by Headjack** [comment]
Seconded. This has often seemed odd to me, and I sometimes pass cold mornings "failing to fall asleep".

**Comment by Oskar636** [comment]
Duplicate of #12455, as it deals with the same issue but " Resting " expands a little onto it.
